packages/web-pkg/src/components/ItemFilter.vue

```diff
@@ -230,12 +230,6 @@ export default defineComponent({
       background-color: var(--oc-color-background-hover) !important;
     }
   }
-  &-input input {
-    border: 1px solid var(--oc-color-swatch-inverse-muted);
-  }
-  &-checkbox input {
-    border: 2px solid var(--oc-color-swatch-inverse-muted);
-  }
```

Comment:
Without this, the filter checkboxes are hidden in dark mode:

<img width="280" alt="image" src="https://user-images.githubusercontent.com/50302941/223649273-88e6903a-cc3d-470b-af4d-8d0a5da877f6.png">

In light mode, the normal background color as well as the drop background color are white. TBH, I don't know why these colours differ in dark mode and why drops have this (ugly 😅 ) light background.

Reply:
> The icon of the current view mode is displayed in a white color now, which is a bit harder to see than the dark color before.

Stupid / naive me, thinking the tokens referencing values of other tokens would be respected at runtime. That's not the case. Only at build time. So the theme actually **does** need to set color vars for the `contrast` swatch tokens. Grmpf, that's not what I wanted to achieve. But probably the way to go for now if we don't want to rebuild the tokens at runtime when switching / loading themes.

Reply:
> Without this, the filter checkboxes are hidden in dark mode:
> In light mode, the normal background color as well as the drop background color are white. TBH, I don't know why these colours differ in dark mode and why drops have this (ugly 😅 ) light background.

Can be solved by setting an input-border color value in dark mode which is not the same as the secondary background color. :trollface: Did that in this PR. Checkbox needs some more fine tuning. Some of it is done in a PR from @hurradieweltgehtunter